src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/db/Recall.kt

```diff
@@ -199,6 +226,26 @@ data class Recall(
 
   fun recallAssessmentDueDateTime(): OffsetDateTime? = recallEmailReceivedDateTime?.plusHours(24)
 
+  fun prisonerFullName(): FullName =
```

Comment:
A bewildering array of types and methods for combinations of person name properties ... 
Just from the code this looks more like 
`fun prisonerNameOnLicense(): FullName` 
than 
`fun prisonerFullName(): FullName`?